stats/config/utils/README.md

```diff
@@ -0,0 +1,10 @@
+# Config utilities
+
+## `find_free_timeslot.py`
+
+It's a tool to roughly visualize the busyness of update schedule to find a timeslot for some new update group.
+
+### Usage
+
+1. Install dependencies from `requirements.txt`: `pip install -r requirements.txt`
```

Comment:
I had this issue

https://stackoverflow.com/questions/39928710/why-is-there-no-tkinter-distribution-found


so I had to replace it with

```
brew install python-tk
pip install tkcalendar croniter
```

Reply:
fr, ok